### DISCUSSION TITLE
Problems in PBPK model of oral absorption

### DISCUSSION THREAD

**Discussion by MaLuisaEti** [Open-Systems-Pharmacology/Forum#1581] [Models]
Hello everyone,

I am developing a PBPK model for a drug with a narrow window of absorption (absorbed mainly in the upper part of the small intestine). After having a model for iv administration that adjusted very well to the experimental values, I went on to make the model for the oral administration of the same drug.
In the following image I show you what the simulation of the model for oral administration looks like without making any adjustments to the parameters involved in the absorption:

![Urine_shape](https://github.com/Open-Systems-Pharmacology/Forum/assets/70070603/25812f60-c697-4c14-872b-39122e38b15f)

As shown in the image, the shape of the urinary drug excretion curve appears strange as it appears that the drug continues to be eliminated indefinitely.
When adjusting the transcellular intestinal permeability value, it is not possible to optimize the shape of the urinary excretion curve.

![URine_shape_data](https://github.com/Open-Systems-Pharmacology/Forum/assets/70070603/7dec065e-2dee-484b-90e7-29e9b96e2ebc)

I was looking at various exchanges in the forum related to drug absorption and one that was very helpful to me was [#942](https://github.com/Open-Systems-Pharmacology/Forum/discussions/942) where it is recommended to optimize specific intestinal permeability with experimental data on the administration of oral solutions of the drug. I did the same but the shape of the curve still follows the same trend.

I would like to know if there is any way to delimit the section where the absorption of the drug occurs in order to improve the simulation?
My drug has a pKa of 3.5 and it is set that the pH of each section of the GIT is taken into account in its absorption (in any case, by checking and unchecking this item in the simulations I have not seen changes in the resulting simulations).

Thank you in advance
Regards

**Comment by StephanSchaller** [comment]
Dear @MaLuisaEti, to understand the behavior of urinary excretion, you have to look at how it is defined as a kinetic (i.e. the rate equation) and what drives it (what are its dependencies/input parameters).
Renal excretion is a linear function by default and it is depending on plasma concentrations. Thus, if you still have systemic drug concentrations, renal CL will continue.
A flattening. i.e., effectively nonexistent CL, despite significant systemic drug concentration levels, could be caused by tubular reabsorption, i.e., active transport in the kidney, which can push excreted drug from the tubuli back into circulation. It might be high affinity-low capacity as excretion still occurs at higher systemic drug concentrations.

A very prominent example of such a kinetic is glucose. Here the reabsorption with SGLT1/2 is extremely effective.

Best,
Stephan

**Reply by MaLuisaEti** [reply]
Thank you very much for your response @StephanSchaller.

The problem I believe I'm having is related to the systemic concentration of the drug after 4 hours of oral administration. Since the IV model for the same drug fits both plasma concentrations and urinary elimination very well, I consider that the main issue is that I'm not taking something into account in its absorption. That's why I was showing you that, despite adjusting the value of transcellular intestinal permeability, the program is considering that the drug is being absorbed beyond its absorption window (upper part of the small intestine).

Perhaps the most appropriate question would be: which parameters present in the software could I modify so that the model reflects the drug's limited absorption window?

Thank you again,
Ma Luisa
